src/ex4/server/db/migrations/20220624142114-create-item.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Items', {
```

Comment:
```suggestion
    await queryInterface.createTable('items', {
```
table name convention